.gitignore

```diff
@@ -18,6 +18,8 @@ local.properties
 **/*.yml
 **/*.gitkeep
 **/resources
+**/*.jar
+**/*.properties
```

Comment:
I would not exclude this *.properties file, is quite often it might be used in /src/main/resources